docker/Dockerfile

```diff
@@ -1,68 +1,66 @@
-FROM --platform=$BUILDPLATFORM ubuntu:18.04 as build
```

Comment:
@hjpotter92 This some kind of autoformatter?

Reply:
yeah, dockfmt: <https://github.com/jessfraz/dockfmt>